src/components/sectioningConfirm/sectioningConfirm.machine.ts

```diff
@@ -134,7 +135,10 @@ export function createSectioningConfirmMachine() {
         ready: {
           initial: "invalid",
           on: {
-            UPDATE_WORK_NUMBER: { actions: "assignWorkNumber" },
+            UPDATE_WORK_NUMBER: {
+              target: "validating",
+              actions: "assignWorkNumber"
+            },
```

Comment:
Same as described in extractionMachine. Adding validation here is useful because it disables the submit button in the sectioniningConfirm page which is based on the state machine state